README.md

```diff
@@ -1,17 +1,39 @@
-# SOEN341-InsertName
+﻿# SOEN341-InsertName
 ### Authors
-* Simon Lim (tech lead) | 
-* Émilie Martin | @ emilie-martin
-* Daniela Venuta | @ daniela-venuta
-* Minh-Tam Do |
-* Luc Nguyen |
-* Karin Kazarian |
+* Simon Lim (tech lead) | @nasaku898
+* Émilie Martin | @emilie-martin
+* Daniela Venuta | @daniela-venuta
+* Minh-Tam Do | @mint47
+* Luc Nguyen | @1999lucnguyen
+* Karin Kazarian | @Karin-kazarian
+
+### Description
+This is a project partial fulfillement of the requirements for the course SOEN341 at Concordia University.
```

Comment:
I think I would replace the contents of `Description` with what you currently have in `Objectives` as the goal of this project is to code a replica of Instagram with it's core functionalities.